src/packaging/dmlinux/Makefile

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright Red Hat
+#
+
+SRC_DIR = ../..
+include ../defines
+
+RPMBUILD      = rpmbuild
+RPMBUILD_FLAG = -bs
+BUILD_DIR    ?= $(realpath .)/build
+WORK_DIR     ?= $(realpath .)/work
+PACKAGE       = kvdo-$(VDO_VERSION)
+PREPARED_DIR  = $(WORK_DIR)/$(PACKAGE)
+SOURCES       = $(BUILD_DIR)/SOURCES
+
```

Comment:
Where does VDO_VERSION come from, here? It's not really defined upstream (other than the kernel version itself), so I want to make sure we're not relying on something irrelevant.

Reply:
I believe it comes from the include of the defines file